subscriptions/subscriptions.go

```diff
@@ -91,6 +98,15 @@ func (p *Subscriptions) HasTxPermission(
 	org *db.Organization,
 	user *db.User,
 ) (bool, error) {
+	if org == nil {
+		return false, fmt.Errorf("organization is nil")
+	}
+
+	// Check if the organization has a subscription
+	if org.Subscription.PlanID == 0 {
+		return false, fmt.Errorf("organization has no subscription plan")
```

Comment:
PlanID 0 is a valid plan id. The check with 0 here does not bring the result you expected